benchmarks/kernels/benchmark_paged_attention.py

```diff
@@ -21,6 +21,7 @@ def main(
     use_alibi: bool,
     block_size: int,
     dtype: torch.dtype,
+    use_fp8_kv_cache: bool,
```

Comment:
Please use a string option instead of bool.
```suggestion
    kv_cache_dtype: Optional[str] = None,
```